src/actions/absoluteAction.ts

```diff
@@ -0,0 +1,47 @@
+module Plywood {
+  export class AbsoluteAction extends Action {
+    static fromJS(parameters: ActionJS): AbsoluteAction {
+      return new AbsoluteAction(Action.jsToValue(parameters));
+    }
+
+    constructor(parameters: ActionValue) {
+      super(parameters, dummyObject);
+      this._ensureAction("absolute");
+    }
+
+    public getOutputType(inputType: string): string {
+      this._checkInputType(inputType, 'NUMBER');
+      return 'NUMBER';
+    }
+
+    protected _getFnHelper(inputFn: ComputeFn): ComputeFn {
+      return (d: Datum, c: Datum) => {
+        var v = Math.abs(inputFn(d, c));
+        return v;
+      }
+    }
+
+    protected _performOnLiteral(literalExpression: LiteralExpression): Expression {
+      if (literalExpression.equals(Expression.ZERO)) {
+        return Expression.ZERO;
+      }
```

Comment:
There is no need to implement this, in fact you can remove this function and it will work just as well.
If the input is a literal and the action has no expression then it will be auto simplified.
In general with simplifications tests it is better to start with a test and then see if any extra work is needed.